backend/app.py

```diff
@@ -24,34 +26,41 @@
 }
 CACHE_TIMEOUT = 1 if app.config['DEBUG'] else os.getenv('CACHE_TIMEOUT', 600)
 
-KEEP_PROPERTIES = ["TYPE", "CATEGORIE", "LIEN", "TITRE", "PRODUCTEUR", "STATUT D'OUVERTURE", "DATE ESTIMÉE", "MINISTÈRE DE TUTELLE"]
-
-@app.route("/api/inventaire")
-@cache.cached(timeout=CACHE_TIMEOUT, query_string=True)
-def inventaire():
-    url = f"{NOTION_API}{NOTION_BASE}/query"
-    payload = {
-        "page_size": 100,
-        "filter": {
-            "property": "PUBLIC",
-            "checkbox": {
-                "equals": True
-            }
-        },
-    }
-    start_cursor = request.args.get("start_cursor")
-    if start_cursor:
-        payload["start_cursor"] = start_cursor
-    r = requests.post(url, json=payload, headers=HEADERS)
+def get_value(property):
+    if property['type'] == 'title':
+        return property['title'][0]['plain_text'].strip()
+    if property['type'] == 'rich_text':
+        return property['rich_text'][0]['plain_text'].strip()
+    elif property['type'] == 'url':
+        return property['url'].strip() if property['url'] is not None else None
+    elif property['type'] == 'select':
+        return property['select']['name'].strip()
+    else:
+        raise RuntimeError(f"Unknown property type {property['type']}")
+    
+def fetch_database(id: str, properties: List[str]) -> List[dict]:
+    # We do not paginate the list, because it should stay below 100 but we could add it
```

Comment:
I'm not sure we'll stay under 100 elements in HVD table :/

Reply:
Done in b396230